### PR TITLE
Catch LockError in locks.unlock()

### PR DESCRIPTION
**Pull request by locriandev**

### CODE REVIEW

File: pyartcd/pyartcd/locks.py [hunk 1]
```diff
@@ -124,9 +124,12 @@ async def lock(self, resource, *args, **kwargs):
...
     async def unlock(self, lock):
-        self.logger.info('Releasing lock "%s"', lock.resource)
-        await super().unlock(lock)
-        self.logger.info('Lock released')
+        try:
+            self.logger.info('Releasing lock "%s"', lock.resource)
+            await super().unlock(lock)
+            self.logger.info('Lock released')
+        except LockError:
+            self.logger.warning('Failed releasing lock %s', lock.resource)
```
Comment (thegreyd) [inline pyartcd/pyartcd/locks.py:132]: Let's log the error message from the exception, and raise again?

Comment (locriandev) [reply pyartcd/pyartcd/locks.py:132]: The LockError doesn't quite carry any useful information, I've tried in the past but seemed unrelevant

Comment (thegreyd) [reply pyartcd/pyartcd/locks.py:132]: But why not raise again, when this happens, as it would previously

Comment (locriandev) [reply pyartcd/pyartcd/locks.py:132]: Right, updated

**Pull request merged**